.github/linters/check_links_config.json

```diff
@@ -1,3 +1,8 @@
 {
+    "ignorePatterns": [
+        {
+          "pattern": "^https://www.microsoft\\.com.*"

```

Comment:
do we also need to escape the dot after www?

Reply:
Yeah we do. Fixed.